<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control</class>
 <widget class="QWidget" name="Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(42, 39, 41);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>6</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Размер лабиринта:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="path_button">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>266</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid black;
   border-radius: 0px;
   font: 20pt;
   background-color: rgb(253, 159, 9);
}
QPushButton:pressed {
    background-color: rgb(202, 125, 6);
    border: 2px solid black;
}</string>
   </property>
   <property name="text">
    <string>Найти путь</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="col_box">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>36</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 14pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Колонки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>176</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 14pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Начало:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_button">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>96</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid black;
   border-radius: 0px;
   font: 20pt;
   background-color: rgb(253, 159, 9);
}
QPushButton:pressed {
    background-color: rgb(202, 125, 6);
    border: 2px solid black;
}</string>
   </property>
   <property name="text">
    <string>Сгенерировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="file_button">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>380</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid black;
   border-radius: 0px;
   font: 20pt;
   background-color: rgb(253, 159, 9);
}
QPushButton:pressed {
    background-color: rgb(202, 125, 6);
    border: 2px solid black;
}</string>
   </property>
   <property name="text">
    <string>Загрузить лабиринт</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="end_y_box">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>476</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid black;
   border-radius: 0px;
   font: 20pt;
   background-color: rgb(253, 159, 9);
}
QPushButton:pressed {
    background-color: rgb(202, 125, 6);
    border: 2px solid black;
}</string>
   </property>
   <property name="text">
    <string>Сохранить лабиринт</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="row_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="start_x_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>239</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>36</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 14pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Строки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>146</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Нахождение пути:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>204</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 14pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Колонки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>234</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 14pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Строки</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="end_x_box">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>239</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>176</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 14pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Конец:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="start_y_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QLabel" name="maze_size">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>340</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 14pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>310</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Текущий размер лабиринта:</string>
   </property>
  </widget>
  <widget class="QLabel" name="file_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 18pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>430</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Файл:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
